ds_judgements_public_ui/templates/includes/recent_judgments.html

```diff
@@ -2,8 +2,13 @@
   <div class="recent-judgments__container">
     <h1 id="recently-published-judgments" class="recent-judgments__header">Recently published judgments</h1>
     {% include "includes/judgment_listing.html" with judgments=recent_judgments %}
-    <p class="judgment-listing__more-info">
-      <a href="{% url 'search' %}">See all recent judgments</a>
-    </p>
+    <div class="container py-2">
+      <a class="recent-judgments__cta-button"
```

Comment:
The centring on this button is a bit off for some reason:

<img width="1239" alt="Screenshot 2024-12-12 at 14 10 01" src="https://github.com/user-attachments/assets/b6050b25-291b-4400-bf3e-e5810f8e0a05" />